apache/yunikorn-core
# [YUNIKORN-869] Improve coverage of node_collection.go

### What is this PR for?
Add some case to cover these functions
1. GetIterator
2. GetNodeSortingPolicy
3. SetSodeSortingPolicy


### What type of PR is it?

### Todos

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-869

### How should this be tested?
Use "make test" command in yonikorn repo and then use "go tool cover --func=coverage.txt" to check the result

### Screenshots (if appropriate)
<img width="1063" alt="截圖 2022-04-13 下午7 28 29" src="https://user-images.githubusercontent.com/45888688/163170682-1ed52964-83ce-41eb-b8ae-317efc0fd3a5.png">
### Questions:
